client/components/App.js

```diff
@@ -15,7 +15,8 @@ const App = props => {
   return <div className="callout primary">
     <BrowserRouter>
       <Switch>
-        <Route exact path="/pet-types" component={PetTypesPage} />
+        <Route exact path ="/pet-types" component={PetTypesPage} />
+        <Redirect from ="/" push to = "/pet-types" />
```

Comment:
Unfortunately, I'm not sure `Redirect` works like this -- did you test it out? I'm fairly certain we can only effectively put `Route`s inside of our `Switch` here.

If you confirm you tested it and it's working fine, let me know.

If not, you may need something like:
```
<Route exact path="/" render={<Redirect to="/pet-types" />}/>
```

Notably and _arguably even better_, you could also just handle this on the backend by adding the redirect there, and skip the frontend altogether!

**Update:** -- I see I'm conflicting with Rocky's feedback on the redirect! I apologize for that! These things will sometimes happen when there are multiple senior devs on a team 😉 With that, I leave your preferred approach to you -- just triple-check and make sure it's all working as expected with the Router syntax :)